### DISCUSSION TITLE
Image based search

### DISCUSSION THREAD

**Discussion by Zulqurn** [MacPaw/OpenAI#16] [Ideas]
I want to search something based on image. Say if I search a particular type of shoes with image I shall get the location of the places it is available in my vicinity. Is that possible with this openAI SPM?

**Comment by Krivoblotsky** [comment]
Hey, @Zulqurn!
According to the most recent OpenAI [presentation](https://www.youtube.com/watch?v=outcGtbnMuQ), they haven't released image-based requests yet, so it's impossible out of the box yet. 
However, you can try using some object detection models that are available publicity, for example [YOLO](https://pjreddie.com/yolo/), or you can try to train your own using CreateML.
